### PR TITLE
[FIX] Increase wait to 5 blocks

### PR DESCRIPTION
**Pull request by adibas03**

### CODE REVIEW

File: app/components/ScheduleWizard/ScheduleWizard.js [hunk 1]
```diff
@@ -188,10 +188,9 @@ class ScheduleWizard extends Component {
       }
     } catch (error) {
       showNotification('Transaction cancelled by the user.', 'danger', 4000);
+      this.scheduleBtn.innerHTML = 'Schedule';
+      document.body.className = originalBodyCss;
     }
-
-    this.scheduleBtn.innerHTML = 'Schedule';
-    document.body.className = originalBodyCss;
```
Comment (Bagaric) [inline app/components/ScheduleWizard/ScheduleWizard.js]: Please return this to it's original spot. It breaks error management when the use schedules a transaction at a point that is too early.

Comment (Bagaric) [reply app/components/ScheduleWizard/ScheduleWizard.js]: Is there a reason this was moved?

Comment (Bagaric) [reply app/components/ScheduleWizard/ScheduleWizard.js]: https://screencast.com/t/LKgTW5DpnNN

Comment (adibas03) [reply app/components/ScheduleWizard/ScheduleWizard.js]: It caused an error of trying to set innerHtml after navigating to awaiting

Comment (Bagaric) [reply app/components/ScheduleWizard/ScheduleWizard.js]: Ok. We need a different solution to the error then. Fixing one error by causing another is not the way to go.

Comment (Bagaric) [reply app/components/ScheduleWizard/ScheduleWizard.js]: I suggest removing this change from this PR and creating a new task/PR that will deal with fixing this properly.

**Comment by adibas03** [comment]
@Bagaric isssue resolved

**Pull request merged**